include/iceflow/consumer-tlv.hpp

```diff
@@ -193,23 +221,30 @@ class ConsumerTlv {
           m_names[interestUri].push_back(manifestName);
         }
       } break;
+        // JSON carries the analyzed results of the compute function of the
+        // upstreams.
       case Json: {
         std::vector<std::string> strs;
         boost::split(strs, interest.getName().toUri(), boost::is_any_of("/"));
+
+        // Pushing the Json Data to the input queue of the consumer
         addBlockToInputQueue(Block(data.getContent(), data.getContentType()));
```

Comment:
I would introduce a variable to make this comment obsolete.

```suggestion
        auto jsonData = Block(data.getContent(), data.getContentType());
        addBlockToInputQueue(jsonData);
```